iam/iam-gcp/src/main/java/com/salesforce/multicloudj/iam/gcp/GcpIam.java

```diff
@@ -0,0 +1,262 @@
+package com.salesforce.multicloudj.iam.gcp;
+
+import com.google.api.gax.rpc.ApiException;
+import com.google.auto.service.AutoService;
+import com.google.cloud.iam.admin.v1.IAMClient;
+import com.google.iam.admin.v1.CreateServiceAccountRequest;
+import com.google.iam.admin.v1.ServiceAccount;
+import com.google.iam.v1.Binding;
+import com.google.iam.v1.GetIamPolicyRequest;
+import com.google.iam.v1.Policy;
+import com.google.iam.v1.SetIamPolicyRequest;
+
+import com.salesforce.multicloudj.common.exceptions.InvalidArgumentException;
+import com.salesforce.multicloudj.common.exceptions.SubstrateSdkException;
+import com.salesforce.multicloudj.common.exceptions.UnSupportedOperationException;
+import com.salesforce.multicloudj.common.exceptions.UnknownException;
+import com.salesforce.multicloudj.common.gcp.CommonErrorCodeMapping;
+import com.salesforce.multicloudj.common.gcp.GcpConstants;
+import com.salesforce.multicloudj.iam.driver.AbstractIam;
+import com.salesforce.multicloudj.iam.model.CreateOptions;
+import com.salesforce.multicloudj.iam.model.PolicyDocument;
+import com.salesforce.multicloudj.iam.model.TrustConfiguration;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@SuppressWarnings("rawtypes")
```

Comment:
I think we don't need this. Also noticed that in AbstractIAM, we don't need the generic type T 

`public abstract class AbstractIam<T extends AbstractIam<T>> implements Provider, Identity`

It's un-used.